pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicPoliciesTest.java

```diff
@@ -3577,9 +3579,13 @@ private void clearTopicPoliciesCache() {
         Map<TopicName, TopicPolicies> globalPoliciesCache =
                 WhiteboxImpl.getInternalState(topicPoliciesService, "globalPoliciesCache");
 
+        Map<NamespaceName, CompletableFuture<SystemTopicClient.Reader<PulsarEvent>>> readerCaches = WhiteboxImpl
+                .getInternalState(topicPoliciesService, "readerCaches");
```

Comment:
Don't use reflection, just expose the field's visibility for tests.

I know existing code has used reflection as well, it's definitely a result of bad quality reviews, but please don't go far in a wrong way.